src/config.ts

```diff
@@ -37,11 +42,10 @@ export const fhirConfig: FhirConfig = {
         // Used in Capability Statement Generation only
         strategy: {
             service: 'OAuth',
-            oauthUrl:
-                process.env.OAUTH2_DOMAIN_ENDPOINT === '[object Object]' ||
-                process.env.OAUTH2_DOMAIN_ENDPOINT === undefined
-                    ? 'https://OAUTH2.com'
-                    : process.env.OAUTH2_DOMAIN_ENDPOINT,
+            oauth: {
+                authorizationUrl: OAuthUrl,
+                tokenUrl: OAuthUrl,

```

Comment:
@rsmayda I was unsure which `url` to use for the `tokenUrl`. I used the same `url` as for `auth`. What are your thoughts?

Reply:
Since we are using Cognito here we can follow Cognito rules: 
```suggestion
                tokenUrl: `${OAuthUrl}/oauth2/token`,

```
- https://docs.aws.amazon.com/cognito/latest/developerguide/token-endpoint.html
- https://docs.aws.amazon.com/cognito/latest/developerguide/authorization-endpoint.html